packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -116,7 +116,10 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
       this.setState({
         value: newProps.value,
         errorMessage: ''
-      } as ITextFieldState);
+      } as ITextFieldState,
+        () => {
+          this._adjustInputHeight();
```

Comment:
>_adjustInputHeight [](start = 15, length = 18)

Can you please add a change file with "patch" for textfield

Reply:
run "npm run change" from the root

---
In reply to: [167708512](https://github.com/OfficeDev/office-ui-fabric-react/pull/3942#discussion_r167708512) [](ancestors = 167708512)

Reply:
@manishgarg1 when I run that command it prints the following:

```
Starting "rush change"

Target branch is origin/master
No change file is needed.
```

What should I do?

Reply:
@nero120  Try `npm run change -- -b upstream/master` (rush is comparing your changes to master, but since your changes were committed to your master branch, this will compare it to Fabric's master)

Reply:
@erichdev nope:

```
ERROR: Command failed: git diff upstream/master... --dirstat=files,0
       fatal: ambiguous argument 'upstream/master...': unknown revision or path not in the working tree.
       Use '--' to separate paths from revisions, like this:
       'git <command> [<revision>...] -- [<file>...]'
```

I confess to not understanding why this doesn't 'just work', I haven't done anything particularly unique here - doesn't everyone submit a pull request from their fork's master branch?

Reply:
"upstream" is a remote name. Do you have a git remote pointing to officedev? You can add it like so:

```bash
git remote add upstream https://github.com/OfficeDev/office-ui-fabric-react.git
```

Reply:
Thanks @dzearing, but after running your command then `npm run change -- -b upstream/master` I get the error:

```
Starting "rush change"

Target branch is upstream/master
fatal: ambiguous argument 'upstream/master...': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Is there some other way I can create this file or can you accept the pull request without it?